exist-core/src/main/resources-filtered/org/exist/system.properties

```diff
@@ -24,6 +24,7 @@ vendor=eXist-db Project
 vendor-url=http://exist-db.org
 product-name=eXist
 product-version=${project.version}
-product-build=${build-tstamp}
+product-build=${maven.build.timestamp}
 git-branch=${build-version}
 git-commit=${build-commit}
+git-commit-timestamp=${build-tstamp}
```

Comment:
confusing variable name :-/

Reply:
<img width="1063" alt="image" src="https://user-images.githubusercontent.com/1700062/200361552-2a79ffe3-f421-4921-a049-9b5dbd92abfc.png">
are the others OK?
wondering who is setting the variable?

Reply:
@dizzzz Thanks for noticing those other files - I have amended the PR to include additional changes to those files.